jupyter_firefly_extensions/__init__.py

```diff
@@ -36,13 +37,42 @@ def initialize(self, notebook_dir, firefly_url):
         self.firefly_url = firefly_url
 
     def get(self):
+        log = logging.getLogger(__name__)
```

Comment:
I recommend to move this to the top level (after the imports) so the logger can be used in other functions. Also the usual naming is `logger`.

Reply:
updated